src/shared/components/HomeToggle/HomeToggle.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import React, { useState } from 'react';
+import * as style from "./HomeToggle.css";
+import clsx from 'clsx';
+
+const HomeToggle = () => {
+
+  const TOGGLE_OPTIONS = {
+    WATCHA_HOME: '왓챠홈',
+    MAGAZINE: '매거진',
+  };
+
+  const [selected, setSelected] = useState<string>(TOGGLE_OPTIONS.WATCHA_HOME);
+
+  const handleToggle = (value: string) => {
+    setSelected(value);
+  };
+
+  const handleStopClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    e.stopPropagation();
+  }
+
+  return (
+    <div className={style.toggleWrapper}>
+      <div className={style.toggleContainer}>
+          <p 
+            className={style.toggleText({ active: selected === TOGGLE_OPTIONS.WATCHA_HOME })}
+            onClick={() => handleToggle(TOGGLE_OPTIONS.WATCHA_HOME)}
+            >왓챠홈
```

Comment:
```suggestion
            >{TOGGLE_OPTIONS.WATCHA_HOME}
```
**Request**: 여기도 상수로 사용해야 만약 옵션 내용이 바뀌었을 때 관리하기 좋을 것 같아용